ui/hooks/useHideFiatForTestnet.ts

```diff
@@ -0,0 +1,14 @@
+import { useSelector } from 'react-redux';
+import { getShowFiatInTestnets, getCurrentChainId } from '../selectors';
+import { TEST_NETWORK_IDS } from '../../shared/constants/network';
+
+/**
+ * Returns true if the fiat value should be hidden for testnet networks.
+ *
+ * @returns boolean
+ */
+export const useHideFiatForTestnet = (): boolean => {
+  const showFiatInTestnets = useSelector(getShowFiatInTestnets);
+  const chainId = useSelector(getCurrentChainId);
```

Comment:
To support multi-chain, should we at least optionally accept the `chainId` as a property rather than adding more reliance on the global network?

Reply:
Good call, added this as a parameter to respect the given `chainId`

Reply:
Done in https://github.com/MetaMask/metamask-extension/pull/25167/commits/07411341fe588535c04301befa1bc358135e0ced